pkg/drawio/layout.go

```diff
@@ -26,6 +26,9 @@ import (
 
 const (
 	minSize         = 10
+	twoMinSize      = 20
+	threeMinSize    = 20
```

Comment:
`twoMinSize` and `threeMinSize` are not being used.

Reply:
fixed